### PR TITLE
redo slug for urls

### PR DESCRIPTION
**Pull request by jacoor**
This is slight change for slugs.
I realized:
- we don't need uniqueness
-  all we need is `<slug>-<id>` in urls

So basically 

```python
# urls.py
path('<slug:slug>-<int:pk>/', NoteDetailView.as_view(), name='note_details'),
```
and in templates 
```
{% url "product" product.slug product.id %}
```
Django is smart enough to handle if slug ends with numbers even. 
I used https://github.com/jacoor/django-w-godzine/tree/master/notes for testing as we don't have any urls in Dshop yet. 

Oh, and one test is dropped since at this point is not valid.

### CODE REVIEW

File: Dshop/apps/products_catalogue/tests.py [hunk 1]
```diff
@@ -15,10 +15,3 @@ def test_create_category_with_parent():
     parent_category = Category.objects.create(name='Parent Category', is_active=True)
     child_category = Category.objects.create(name='Child Category', is_active=True, parent=parent_category)
     assert child_category.parent == parent_category
-
-
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/tests.py:19]: The test below is not valid anymore and there is no case we could test now.

File: Dshop/apps/products_catalogue/models.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 class CatalogueItemModel(models.Model):
     name = models.CharField(max_length=200)
-    slug = models.SlugField(max_length=200)
+    slug = models.SlugField(max_length=200, editable=False)
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/models.py:7]: Editable=false makes field invisible in admin.

File: Dshop/apps/products_catalogue/tests.py [hunk 3]
```diff
@@ -15,10 +15,3 @@ def test_create_category_with_parent():
...
-@pytest.mark.django_db
-def test_different_slugs_for_same_category_name():
-    category1 = Category.objects.create(name='Main Category', is_active=True)
-    category2 = Category.objects.create(name='Main Category', is_active=True)
-    assert category1.slug != category2.slug
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/tests.py:24]: This test is not valid anymore and there is no case we could test, at least now.

**Review by memorisanka — APPROVED** [review]
I am not sure if it will be 100% unique (I didn't use slug), but I think it will be OK for us for now :)

**Comment by jacoor** [comment]
The magic is it doesn't have to be unique. Slug + id will be unique.

**Pull request merged**